File: pgm/ABC_Java/ABC_218_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final int N = Integer.parseInt (reader.readLine());
        final String S = reader.readLine();
        final char circle = 'o';
        final String Yes = "Yes";
        final String No = "No";
        if (S.charAt (N - 1) == circle) {
            System.out.println (Yes);
        } else {
            System.out.println (No);
        }
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_218_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.LocalTime
// import java.time.format.DateTimeFormatter
// import java.util.Scanner
// import java.util.StringTokenizer

// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val circle: Char = 'o'
const val one: Int = 1

fun main() {
    val N: Int = readln().toInt()
    val S: String = readln()
    if (S[N - one] == circle) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}